docker/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN install2.r --error \
 
 RUN Rscript -e 'remotes::install_github(c("mrc-ide/dust", "mrc-ide/mcstate"))'
 
-RUN Rscript -e 'remotes::install_github("mrc-ide/dust")'
+RUN Rscript -e 'remotes::install_github("mrc-ide/dust@v0.7.5")'
```

Comment:
Should this version be pinned?

Reply:
not any more!

Reply:
oh, hang on that's there to invalidate the docker build cache. I'll do something else that will be less annoying in future